_data/blogs.yml

```diff
@@ -37,6 +37,11 @@
   title: Interview with a Teacher from Doon International School
   img: yashkumarverma-blog02.jpg
   author_name: Yash Kumar Verma
+  
+- url: https://ziadelkhattabi.wordpress.com/2017/12/11/first-blog-post/
+  title: BETWEEN THE COLORED LINES
+  img: computer-programming.jpeg
```

Comment:
Please upload this image

Reply:
ok

Reply:
When I try to upload  it, it said that uploads are disabled

Reply:
Are you uploading in your forked repository?